modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -95,7 +95,7 @@ private function check_iframe_cookie_setting() {
 			return;
 		}
 
-		if ( ! $_GET['calypsoify_cookie_check'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		if ( ! isset( $_GET['calypsoify_cookie_check'] ) || ! $_GET['calypsoify_cookie_check'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
It would probably be simpler to use `empty( $_GET['calypsoify_cookie_check'] )`, but that's not a blocker.

Reply:
I always forget about `empty` taking care of `isset` for us. Good idea. e17f7a016